.github/workflows/deploy.yml

```diff
@@ -2,47 +2,63 @@ name: DeployCI
 
 on:
   push:
-    branches: [ master, deploy-work ]
+    branches: [master, deploy-work]
 
   workflow_dispatch:
 
 jobs:
-  build-api:
+  build-and-deploy:
     runs-on: ubuntu-latest
 
-
     steps:
       - uses: actions/checkout@main
-      - run: |
-          sudo apt-get -y install nodejs npm
+
```

Comment:
Add the following step before `Cache node modules`, otherwise cache won't work:
```yaml
      - name: Setup node
        uses: actions/setup-node@v2
        with:
          node-version: '16'
```

Reply:
this step needs to be added